modules/virt-querying-metrics.adoc

```diff
@@ -12,14 +12,17 @@ The following metric descriptions include example Prometheus Query Language (Pro
 These metrics are not an API and might change between versions.
 ====
 
-
 [id="virt-promql-vcpu-metrics_{context}"]
 == vCPU metrics
 
+.Prerequisite
+
+* The vCPU metric requires the `schedstats=enable` kernel argument applied to the `MachineConfig` object before it can be used. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler. See the xref:../../post_installation_configuration/machine-configuration-tasks.adoc#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks[{product-title} machine configuration tasks] documentation for more information on applying a kernel argument.
```

Comment:
We cannot use xrefs in modules. I think that is why this paragraph was placed at the assembly level in the existing documentation.

You will need to add the link in the Additional resources section in the assembly and edit the paragraph content accordingly if you decide to keep this content in the module.